config/routes.rb

```diff
@@ -1,8 +1,13 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   root 'welcome#index'
+
   get '/registration', to: 'users#new'
+
   resources :users, only: [:create]
-  get '/dashboard', to: 'dashboard#index'
+
+  resources :dashboard, only: [:index]
```

Comment:
Can we change this back to `get  '/dashboard'`? My understanding is that the `resource` of `resources` refers to database resources, and that it should only be used to create ReSTful routes dealing with database tables. Since `/dashboard` is not ReSTful and there is no database table I believe it should be hand-rolled